config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+
+  get '/movies', to: 'movies#index', as: 'movies'
+  get '/movies/:title', to: 'movies#show', as: 'movie'
```

Comment:
It's totally great as is, just know that you will more often than not (in my -admittedly limited - experience) see :id in the routes for show.  There's a little Rails magic you can tap into if you use the convention of /model/:id in routes. If you're curious to Google or you come across something like `resources :movies` in a routes file, it's generating all of the routes for you and will format the show method with the id in the route.